### DISCUSSION TITLE
How to format prompts in survey plugin

### DISCUSSION THREAD

**Discussion by DEMelnikoff** [jspsych/jsPsych#3070] [Q&A]
Hello!

I'm using the Survey plugin, which is is great, but can't seem to figure out how to format the prompt text for the likert question-type. I'd like to be able to bold a particular world, but when I set the prompt to something like 'I want to bold <b>this</b> word' the tags are not interpreted as such and are just displayed as text. Any advice on how to get the plugin to interpret prompts as html strings?

**Comment by ChristopheBossens** [comment]
From the docs:

> The prompt/question that will be associated with the question's response field. If the question type is "html", then this string is the HTML-formatted string to be displayed. If the question type is "likert-table", the prompt is a general question or title presented above the table.

So it seems that it is not possible for likert questions to provide an html-formatted string as a prompt. One hack that you could use is to have an html question-type above your likert question in which you can format the prompt string. In the likert question you would have to set the prompt to a string with a space (in case of an empty string it seems to output a default name).

**Reply by DEMelnikoff** [reply]
Thanks! I tried to alter the javascript in the plugin itself to allow it to take html strings as prompts but never managed to get it working. This is a good alternative option.

**Reply by vscholz118** [reply]
Hello @DEMelnikoff  and @ChristopheBossens , 
This is exactly the problem I'm currently struggling with but I have a hard time imagining how the suggested solution would look like in actual code (sorry I'm rather new to the plugins actually and still trying to figure everything out). Hence, my question, could you provide an example code snippet for this, that worked for you? This would be really great!
